addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -136,13 +140,13 @@
                 </div>
             </div>
         </section>
-        <t t-call="website_event.modal_ticket_registration"/>
     </t>
 </template>
 
 <!-- Event - Registration -->
+<!-- cta_additional_classes {string} - classes applied to the main CTA   -->
 <template id="registration_template" name="Registration">
-    <button t-if="event.event_registrations_open" type="button" data-bs-toggle="modal" data-bs-target="#modal_ticket_registration" class="btn btn-primary w-100">Register</button>
+    <button t-if="event.event_registrations_open" type="button" data-bs-toggle="modal" data-bs-target="#modal_ticket_registration" t-attf-class="btn btn-primary {{cta_additional_classes}}">Register</button>
     <div t-if="toast_message" class="o_wevent_register_toaster d-none" t-att-data-message="toast_message"/>
```

Comment:
Note that this is now added on every pages, but looking at the code it doesn't seem to have an impact, it's only assigned when going on the /register page from specific links.